html-css/week1/Nina/portfolio/index.html

```diff
@@ -0,0 +1,177 @@
+      whether it’s building websites or picking up a new language like Danish. When I’m not coding, you’ll
+      probably find me
+      exploring Copenhagen, trying out new recipes, or teaching Chinese to kids on weekends.
+    </p><br>
+    <p>I’m excited to see where this journey in tech will take me, and I’m always up for connecting with people
+      who share a
+      love for creativity and growth!</p>
+  </div>
+</div>
+</section>
+<section id="projects" class="container">
+  <h2>Projects</h2>
+  <div class="projects">
+  <div class="project-left">
+    <h4>My first Project</h4>
+    <iframe height="400" style="width: 100%;"  title="FCC Survey Form"
+      src="https://codepen.io/LinRuoyin/embed/emOrxev" frameborder="no" allowfullscreen="true"></iframe>
+    </div>
+  <div class="project-right"><h4>To be continued...</h4></div>
+</div>
+</div>
+
+</section>
+
+<section id="contact" class="container">
+  <h2>Contact Me</h2>
+  <div class="row">
+    <div class="contact-left">
+      <h3>Contact info</h3>
+      <div><svg class="w-6 h-6 text-gray-800 dark:text-white" aria-hidden="true" xmlns="http://www.w3.org/2000/svg"
+          width="24" height="24" fill="none" viewBox="0 0 24 24">
+          <path stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"
+            d="M11 16v-5.5A3.5 3.5 0 0 0 7.5 7m3.5 9H4v-5.5A3.5 3.5 0 0 1 7.5 7m3.5 9v4M7.5 7H14m0 0V4h2.5M14 7v3m-3.5 6H20v-6a3 3 0 0 0-3-3m-2 9v4m-8-6.5h1" />
+        </svg>
+
+        <a href="mailto:nina.linn@outlook.com" class="">
+          <p>nina.linn@outlook.com</p>
+        </a>
+      </div>
+      <br>
+      <div>
+        <svg class="w-6 h-6 text-gray-800 dark:text-white" aria-hidden="true" xmlns="http://www.w3.org/2000/svg"
+          width="24" height="24" fill="currentColor" viewBox="0 0 24 24">
+          <path
+            d="M7.978 4a2.553 2.553 0 0 0-1.926.877C4.233 6.7 3.699 8.751 4.153 10.814c.44 1.995 1.778 3.893 3.456 5.572 1.68 1.679 3.577 3.018 5.57 3.459 2.062.456 4.115-.073 5.94-1.885a2.556 2.556 0 0 0 .001-3.861l-1.21-1.21a2.689 2.689 0 0 0-3.802 0l-.617.618a.806.806 0 0 1-1.14 0l-1.854-1.855a.807.807 0 0 1 0-1.14l.618-.62a2.692 2.692 0 0 0 0-3.803l-1.21-1.211A2.555 2.555 0 0 0 7.978 4Z" />
+        </svg>
+        <p>+45 71840871</p>
+      </div>
+      <div class="social-links">
+        <a href="https://www.github.com/LinRuoyin" target="_blank">
+          <img src="./imgs/github.png" alt="GitHub" width="48px">
+        </a>
+        <a href="https://www.linkedin.com/in/ruoyin-lin-nina" target="_blank">
+          <img src="./imgs/linkedin.png" alt="LinkedIn" width="48px">
+        </a>
+      </div>
+    </div>
+    <div class="contact-right">
+      <form action="" name="submit-to-google-sheet" id="myform">
+        <input type="text" name="Name" placeholder="Your Name" required>
+        <input type="email" name="Email" placeholder="Your Email" required>
```

Comment:
A good accessibility practice is to add `<label>` tags and associate them with the `<input>` fields. Placeholder text disappears as soon as the user starts typing inside of a field, and screen readers (devices used by people with visual disabilities) don't read `placeholder`. Here is an example on how to add `<label>` and associate the tags:
```suggestion
        <label for="your-name">Your Name</label>
        <input type="text" id="your-name" name="your-name" placeholder="Your Name" required>
        <label for="your-email">Your Email</label>
        <input type="email" id="your-email" name="your-email" placeholder="Your Email" required>
```